examples/simple-platform/run.md

```diff
@@ -0,0 +1,273 @@
+need to create a new GitHub OAuth app and configure it to connect to the
+`web-auth-endpoint` node.
+
+1. Go to [github.com](https://github.com/) and log into your account.
+
+1. Navigate to Settings > Developer Settings > OAuth Apps.
+
+1. Click the **New OAuth App** button to create a new OAuth app and configure
+it.
+
+1. In the **Application name** field, give the app a name of your choice,
+e.g. `Interbit Test App`.
+
+1. Fill in the **Homepage URL** field with any URL. Any URL will do since we
+are running the example locally.
+
+1. Fill in the **Authorization callback URL** field with
+`http://localhost:8888/oauth/github`.
+
+   Note: The callback URL is case sensitive and the port specified is for
+running the `web-auth-endpoint` node locally.
+
+1.  Click the **Register application** to finish creating the OAuth app.
+
+    Leave this browser tab open as we are going to copy values from this tab in
+subsequent steps.
+
+
+### Clone the `interbit` repository and install `interbit-cli`
+
+1. Open a terminal and clone the `interbit` repository from GitHub:
+
+    ```sh
+    git clone https://github.com/interbit/interbit.git
+    ```
+
+1. Install `interbit-cli`:
+
+    ```sh
+    npm i -g interbit-cli
+    ```
+
+    This will install the [Interbit CLI](/reference/interbit-cli/README.md)
+globally.
+
+
+### Generate a set of private and public keys for the `platform-deploy` and `web-auth-endpoint` hypervisors
+
+1. Create a `secrets` directory at the root of the `interbit` repository.
+
+   The `secrets` directory is included in this repository's `.gitignore` file
+and is not tracked by git. We don't want to commit the files we are going to
+add here because they include private keys for our Interbit nodes.
+
+1. Change the current working directory to the `secrets` directory:
+
+    ```sh
+    cd secrets
+    ```
+
+1. Generate an Interbit key pair for `platform-deploy` and `web-auth-endpoint`,
+respectively, with:
+
+    ```sh
+    interbit keys --filename platform-deploy-keys.json
+    interbit keys --filename web-auth-endpoint-keys.json
+    ```
+
+  This creates two new files at the root of the `interbit` repository with your
+  public and private key pairs.
+
+
+### Set up environment variables for `platform-deploy`
+
+1. Create the file `platform-deploy.sh` in the `secrets` directory with the
+following content:
+
+    ```bash
+    #!/bin/bash
+    # Secrets for Accounts app GitHub OAuth chain
+    export GITHUB_CLIENT_ID=""
+    export GITHUB_CLIENT_SECRET=""
+    export GITHUB_REDIRECT_URL=""
+    export OAUTH_CALLBACK_URL="http://localhost:3025/account/oauth/gitHub"
+    # Key pair for the platform node
+    export PUBLIC_KEY=""
+    export PRIVATE_KEY=""
+    # Peer list override
+    export PORT=5025
+    export CONNECT_TO_PEERS="localhost:8888"
+    ```
+
+1. Copy the values for the `GITHUB_CLIENT_ID`, `GITHUB_CLIENT_SECRET`, and
+`GITHUB_REDIRECT_URL` fields from the GitHub OAuth app you just created (in the
```

Comment:
This step needs updating:
- There are no `GITHUB_*` fields on the GitHub OAuth app page, so we need to tell people to `copy from field X and paste in field Y`.
- There is no `redirect URL` field on the GitHub OAuth page. I used the `Authorization Callback URL`, but I'm not sure if that's what's intended.

Reply:
That's what's intended. I updated the docs so that the copy/paste correspondence is explicitly stated.